TimeLog.TLP.API.XML

```diff
@@ -0,0 +1,7720 @@
+            </summary>
+            <param name="isSecure">Decide if the service is used over https or http</param>
+        </member>
+        <member name="M:TimeLog.TLP.API.Invoicing.V1_1.InvoicingService.GetInvoiceByID(System.Int32,TimeLog.TLP.API.Proxy.Security.V1_3.SecurityToken)">
+            <summary>
+            Gets an invoice based on its ID
+            </summary>
+            <param name="id">Identifier of the invoice (int)</param>
+            <param name="token">Security token associated with the request (SecurityToken)</param>
+            <returns>A invoice matching the identifier</returns>
+        </member>
+        <member name="M:TimeLog.TLP.API.Invoicing.V1_1.InvoicingService.GetInvoiceByGUID(System.Guid,TimeLog.TLP.API.Proxy.Security.V1_3.SecurityToken)">
+            <summary>
+            Gets an invoice based on its GUID
+            </summary>
+            <param name="guid">Identifier of the invoice (Guid)</param>
+            <param name="token">Security token associated with the request (SecurityToken)</param>
+            <returns>A invoice matching the identifier</returns>
+        </member>
+        <member name="M:TimeLog.TLP.API.Invoicing.V1_1.InvoicingService.GetInvoiceSpecificationPDF(System.Int32,TimeLog.TLP.API.Proxy.Security.V1_3.SecurityToken)">
+            <summary>
+            Gets an invoice specificate based on the invoice ID
+            </summary>
+            <param name="invoiceID">ID of the invoice (int)</param>
+            <param name="token">Security token associated with the request (SecurityToken)</param>
+            <returns>A byte array containing the invoice specification pdf</returns>
+        </member>
+        <member name="M:TimeLog.TLP.API.Invoicing.V1_1.InvoicingService.GetCreditNoteByID(System.Int32,TimeLog.TLP.API.Proxy.Security.V1_3.SecurityToken)">
+            <summary>
+            Gets a credit note based on its ID
+            </summary>
+            <param name="id">Identifier of the credit note (int)</param>
+            <param name="token">Security token associated with the request (SecurityToken)</param>
+            <returns>A credit note matching the identifier</returns>
+        </member>
+        <member name="M:TimeLog.TLP.API.Invoicing.V1_1.InvoicingService.GetCreditNoteByGUID(System.Guid,TimeLog.TLP.API.Proxy.Security.V1_3.SecurityToken)">
+            <summary>
+            Gets a credit note based on its GUID
+            </summary>
+            <param name="guid">Identifier of the credit note (Guid)</param>
+            <param name="token">Security token associated with the request (SecurityToken)</param>
+            <returns>A credit note matching the identifier</returns>
+        </member>
+	    <member name="M:TimeLog.TLP.API.Invoicing.V1_1.InvoicingService.GetCreditNotesModifiedInPeriodPaged(System.DateTime,System.DateTime,System.Int32,System.Int32,TimeLog.TLP.API.Proxy.Security.V1_3.SecurityToken)">

```

Comment:
Det her er den nye tilføjelse.

Reply:
Hi @NGU-droid. You misplaced the file. It should be placed here `TimeLog.Api.Documentation\Source\TimeLog.TLP.API.XML` overwriting the old version. Please correct and push an update. Thanks.